frontend/style.md

```diff
@@ -157,6 +157,8 @@ export const IMPORT_DATASET = gql`
 
 We have two styles when naming functions: `onXXXX` and `handleXXXX`. Note both are camelCase. We use `onXXXX` for functional props, and we use `handleXXXX` when we are locally declaring the function.
 
+Also we have abbreviation like CSV, JSON and HTML. The abbreviation should be integrated into lowercase or uppercase like `xxxCSV` or `csvXXX`.
```

Comment:
Since you are duplicating, can we condense?
Since both the Function and Variable sections are just about naming (at the moment), maybe it is better to create a `Naming Conventions` section and move both sections there. 

Also, maybe rewrite the above like:
`Abbreviations, like CSV, JSON or HTML, should always be uppercase OR lowercase.`

Then, can you copy a good/bad table from another section and give some examples of good and bad?

Reply:
I fixed it.
[fix](https://github.com/reearth/guides/pull/1/commits/25881a8ca2e2d47dfb03ea5cfbd59413fa2ef9ad)